### DISCUSSION TITLE
Agent Committee Meeting Reminder

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#7205] [Agent Committee]
> Team,
> 
> I know this is (sort of) short notice, but the meeting was already on the calendar, so I thought I would attempt it. I'd like to use tomorrow's Agent meeting time (January 10, 1PM Pacific) to have a broad discussion about Agents and where we want to go with them in Arctos.
> 
> After the Bionomia webinar, Michelle, Dusty and Teresa were thinking about Agents and what could end the current anxiety over the “required” use of verbatim agent when little is known about an agent.
> 
> Please read these before coming to the meeting if you can!
> 
> [Arctos Agents - Functional Requirements](https://docs.google.com/document/d/1o9fvy_cAtwoh3rIwPnG90SDUEexa7lKQL1DTxHqswcw/edit?usp=sharing)
> [Arctos Agents Roadmap](https://docs.google.com/document/d/1tSk344mJsIRYZ-BJcYfyaPEmbrFS-wSqZzJ5dO1E2EU/edit?usp=sharing)
> 
> We hope that we can at least reach an agreement on how to end the current situation and be happy with whatever happens after that.
> 
> If nobody can make this meeting, I'll attempt to create another.
> 
> Adios,

Teresa J. Mayfield-Meyer

**Comment by ewommack** [comment]
Hi @Jegelewicz - I'll try. I've got some organizing to do at the research station, but should hopefully be back on campus before 2pm MT.

**Comment by DerekSikes** [comment]
Teresa,

Can you send me the zoom link & why not always include that link in all
emails for meetings?

-Derek


On Tue, Jan 9, 2024 at 8:02 AM Teresa Mayfield-Meyer <
***@***.***> wrote:

> Team,
>
> I know this is (sort of) short notice, but the meeting was already on the
> calendar, so I thought I would attempt it. I'd like to use tomorrow's Agent
> meeting time (January 10, 1PM Pacific) to have a broad discussion about
> Agents and where we want to go with them in Arctos.
>
> After the Bionomia webinar, Michelle, Dusty and Teresa were thinking about
> Agents and what could end the current anxiety over the “required” use of
> verbatim agent when little is known about an agent.
>
> Please read these before coming to the meeting if you can!
>
> Arctos Agents - Functional Requirements
> <https://docs.google.com/document/d/1o9fvy_cAtwoh3rIwPnG90SDUEexa7lKQL1DTxHqswcw/edit?usp=sharing>
> Arctos Agents Roadmap
> <https://docs.google.com/document/d/1tSk344mJsIRYZ-BJcYfyaPEmbrFS-wSqZzJ5dO1E2EU/edit?usp=sharing>
>
> We hope that we can at least reach an agreement on how to end the current
> situation and be happy with whatever happens after that.
>
> If nobody can make this meeting, I'll attempt to create another.
>
> Adios,
>
> Teresa J. Mayfield-Meyer
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ArctosDB/arctos/discussions/7205>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACFNUM3H6P5KXDZP3HIDOQDYNVZ3NAVCNFSM6AAAAABBTNUP2GVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGA2TCNBWGE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 

+++++++++++++++++++++++++++++++++++
*Derek S. Sikes*, Curator of Insects, Professor of Entomology
University of Alaska Museum (UAM), University of Alaska Fairbanks
1962 Yukon Drive, Fairbanks, AK   99775-6960
***@***.*** phone: 907-474-6278 he/him/his
University of Alaska Museum <https://www.uaf.edu/museum/collections/ento/>
-  search 393,654 digitized arthropod records
<https://arctos.database.museum/uam_ento>
+++++++++++++++++++++++++++++++++++

Interested in Alaskan Entomology? Join the Alaska Entomological
Society and / or sign up for the email listserv "Alaska Entomological
Network" at
http://www.akentsoc.org/contact_us

**Reply by dustymc** [reply]
https://berkeley.zoom.us/j/92544859519
